prov/usnic/src/usdf_ep_dgram.c

```diff
@@ -322,7 +322,7 @@ static struct fi_ops_msg usdf_dgram_ops = {
 	.sendv = usdf_dgram_sendv,
 	.sendmsg = usdf_dgram_sendmsg,
 	.inject = usdf_dgram_inject,
-	.senddata = usdf_dgram_senddata,
+	.senddata = fi_no_msg_senddata,
```

Comment:
All of the endpoints implement the same `fi_ops_msg` interface with the same functions. In this case the `msg` in `fi_ops_msg` does not refer to `EP_MSG`.